doc/changes/latest.inc

```diff
@@ -34,4 +34,4 @@ Bugs
 
 API changes
 ~~~~~~~~~~~
-- None yet
+- :func:`mne.utils._validate_type` can now check if a given object is `"array-like"` (:gh:`11713` by `Clemens Brunner`_)
```

Comment:
Oh. No. I mean, this is such an internal thing, it shouldn't show up in the changelog.

What **should** go into the changelog is the user-visible change that `baseline` can now be any `array-like of length 2` instead of just a tuple

Reply:
I would also suggest to rename this PR accordingly

Reply:
Ah, got it! OK!

Reply:
Although I think it would also be good to announce internal things somewhere, so that devs know what has changed and that they can now perform this new check.

Reply:
Regarding the `baseline` change, technically it could be array-like for MNE < 1.1. This changed in MNE 1.1, but I can't find an entry for that change.